docs/installation-vagrant.rst

```diff
@@ -161,7 +161,15 @@ To enable KumaScript (Kuma's template system):
 #. Change ``KUMASCRIPT_TIMEOUT`` to 600
 #. Click "Save" at the bottom
 
+KumaScript is now enabled. You will also want to import the `KumaScript
+auto-loaded modules`_. You can simply copy & paste them from the production
```

Comment:
The CSS on links causes the whitespace here to be treated as significant and so there's a funny line break. Can you put the entire link on one line? Fixing the CSS looks like a PITA. Sorry.